smart-wallets/max-limit-smart-wallet/max-limit-smart-wallet.pact

```diff
@@ -73,7 +73,7 @@
         "max-custodial-amount": max-custodial-amount
        }))
 
-  (defun custodial-transfer (sender:string receiver:string amount:decimal)
+  (defun custodial-transfer:string (sender:string receiver:string amount:decimal)
```

Comment:
Per our conversation let's change the words custodial and non-custodial to something different to avoid confusion.  Perhaps high-security and low-security, or maybe unlimited and limited?  I'm open to other ideas.